org.adempiere.base/src/org/compiere/process/ServerProcessCtl.java

```diff
@@ -83,7 +85,19 @@ public static ServerProcessCtl process (ProcessInfo pi, Trx trx, boolean managed
 		{
 			try 
 			{ 
-				instance = new MPInstance(Env.getCtx(), pi.getAD_Process_ID(), pi.getTable_ID(), pi.getRecord_ID(), pi.getRecord_UU()); 
+				instance = new MPInstance(Env.getCtx(), pi.getAD_Process_ID(), pi.getTable_ID(), pi.getRecord_ID(), pi.getRecord_UU());
+				// Get PrintFormat
+				MPrintFormat format = (MPrintFormat)pi.getTransientObject();
+				if (format == null)
+					format = (MPrintFormat)pi.getSerializableObject();
+				if (format == null)
+				{
+					log.log(Level.INFO, "No PrintFormat");
+				}

```

Comment:
this code runs for both report and process, therefore, we shouldn't log anything here if format is null.